src/app/register/pages/register/register.component.html

```diff
@@ -64,12 +65,13 @@
           ></app-step-b>
         </mat-step>
         <mat-step [stepControl]="FormGroupStepC" #lastStep>
-          <ng-template matStepLabel i18n="@@register.personalNotificationsTerm"
-            >Notifications and terms</ng-template
+          <ng-template matStepLabel i18n="@@register.VisibilityAndTerm"
```

Comment:
@leomendoza123 I think the property it should be 'register.visibilityAndTerm' and register.securityAndNotifications using camelCase

Reply:
👍